pkg/pod/script.go

```diff
@@ -130,9 +130,18 @@ func convertScripts(shellImageLinux string, shellImageWin string, steps []v1beta
 func convertListOfSteps(steps []v1beta1.Step, initContainer *corev1.Container, placeScripts *bool, breakpoints []string, namePrefix string) []corev1.Container {
 	containers := []corev1.Container{}
 	for i, s := range steps {
+		// Add debug mounts if breakpoints are present
```

Comment:
I move the logic of adding debug script before judging the script make sure it can work well no matter s.Script is empty or not.